src/math-translator.js

```diff
@@ -15,16 +15,21 @@ const MATH_RULES_LOCALES = {
     NO_THOUSAND_SEP: ['ko', 'ps', 'ka'],
     DECIMAL_COMMA: ['cs', 'fr', 'de', 'pl', 'bg', 'nb', 'tr', 'da', 'sr', 'lol',
             'ro', 'nl', 'hu', 'az', 'it', 'pt', 'pt-pt', 'sv', 'el', 'id', 'ka',
-            'ru'],
+            'ru', 'ta'],
     // TODO(danielhollas):remove 'bg' from TIMES_AS_CDOT
     // when \mathbin{.} becomes available for them
-    TIMES_AS_CDOT: ['cs', 'pl', 'de', 'nb', 'sr', 'ro', 'hu', 'sv', 'da', 'bg',
-            'lol'],
-    DIV_AS_COLON: ['cs', 'de', 'bg', 'hu', 'uk', 'da', 'hy', 'pl', 'lol', 'id',
+    TIMES_AS_CDOT: ['cs', 'pl', 'de', 'nb', 'sr', 'ro', 'hu', 'sv', 'da', 'bg'],
+    CDOT_AS_TIMES: ['fr', 'ps', 'pt-pt', 'ta'],
+    DIV_AS_COLON: ['cs', 'de', 'bg', 'hu', 'uk', 'da', 'hy', 'pl', 'it',
             'pt-pt', 'ru', 'nb'],
     SIN_AS_SEN: ['it', 'pt', 'pt-pt'],
     ARABIC_COMMA: ['ps'],
     PERSO_ARABIC_NUMERALS: ['ps'],
+    MAYBE_DIV_AS_COLON: ['id', 'lol'],
+    MAYBE_TIMES_AS_CDOT: ['az', 'bn', 'el', 'hi', 'hy', 'id', 'it', 'ja', 'ka',
```

Comment:
In principle, MAYBE_TIMES_AS_CDOT === MAYBE_CDOT_AS_TIMES, but I made it as a separate rule to preserve symmetry with other rules. But I do not have a strong opinion on this.

Reply:
I added a test to make sure they are equal.